src/main/java/com/stripe/exception/TemporarySessionExpiredException.java

```diff
@@ -18,10 +18,11 @@ private TemporarySessionExpiredException(
   static TemporarySessionExpiredException parse(
       JsonObject body, int statusCode, String requestId, StripeResponseGetter responseGetter) {
     TemporarySessionExpiredException.TemporarySessionExpiredError error =
-        StripeObject.deserializeStripeObject(
-            body,
-            TemporarySessionExpiredException.TemporarySessionExpiredError.class,
-            responseGetter);
+        (TemporarySessionExpiredException.TemporarySessionExpiredError)
+            StripeObject.deserializeStripeObject(
+                body,
+                TemporarySessionExpiredException.TemporarySessionExpiredError.class,
+                responseGetter);
```

Comment:
This is a generated file. When I re-run the codegen, this change gets reverted. Do we know why we needed to add this casting?

cc @prathmesh-stripe, @jar-stripe

Reply:
The casting is needed or else the java compiler complains.  I saw this last night too and assumed it was something to do with the codegen branch not being on the latest spec version, but now that I think about it that should affect this.

Reply:
We will need to update codegen to add this casting https://jira.corp.stripe.com/browse/DEVSDK-2195